benchmark/benchmain/main.go

```diff
@@ -114,34 +114,42 @@ var (
 	}
 )
 
-func unaryBenchmark(startTimer func(), stopTimer func(int32), benchFeatures stats.Features, benchtime time.Duration, s *stats.Stats) int32 {
+func unaryBenchmark(startTimer func(), stopTimer func(uint64), benchFeatures stats.Features, benchtime time.Duration, s *stats.Stats) uint64 {
 	caller, cleanup := makeFuncUnary(benchFeatures)
 	defer cleanup()
 	return runBenchmark(caller, startTimer, stopTimer, benchFeatures, benchtime, s)
 }
 
-func streamBenchmark(startTimer func(), stopTimer func(int32), benchFeatures stats.Features, benchtime time.Duration, s *stats.Stats) int32 {
+func streamBenchmark(startTimer func(), stopTimer func(uint64), benchFeatures stats.Features, benchtime time.Duration, s *stats.Stats) uint64 {
 	caller, cleanup := makeFuncStream(benchFeatures)
 	defer cleanup()
 	return runBenchmark(caller, startTimer, stopTimer, benchFeatures, benchtime, s)
 }
 
-func unconstrainedStreamBenchmark(benchFeatures stats.Features, benchtime time.Duration) (int32, int32) {
+func unconstrainedStreamBenchmark(benchFeatures stats.Features, warmuptime, benchtime time.Duration) (uint64, uint64) {
 	sender, recver, cleanup := makeFuncUnconstrainedStream(benchFeatures)
 	defer cleanup()
-	// Warm up connection.
-	for i := 0; i < 10; i++ {
-		sender(0)
-		recver(0)
+	// Send a request to each stream to let server know about the response type and size.
```

Comment:
Can we delete this now that warmup happens as part of the normal benchmark runs?